pointmatcher/PointMatcher.h

```diff
@@ -155,9 +155,13 @@ struct PointMatcher
 	// eigen and nabo-based types
 	// ---------------------------------
 	
+	//! The dimension of the point clouds that libpointmatcher will process
+	static constexpr Eigen::Index kPointDimension{ 3 };
```

Comment:
I am not sure this is the right place for this constant, it seems to be 3d specific. If possible, I would define it in the point cloud generator.